contracts/src/CollateralRegistry.sol

```diff
@@ -280,6 +285,19 @@ contract CollateralRegistry is ICollateralRegistry {
         require(_amount > 0, "CollateralRegistry: Amount must be greater than zero");
     }
 
+    function updateDebtLimit(uint256 _indexTroveManager, uint256 _newDebtLimit) external onlyGovernor {
+        //limited to increasing by 2x at a time, maximum. Decrease by any amount.
+        uint256 currentDebtLimit = getTroveManager(_indexTroveManager).getDebtLimit();
```

Comment:
will this still work with our trove manager index changes?

Reply:
Yeah it should. In PR #9, I modify the `getTroveManager` function to get trove manager of branchId based on index in activeBranchIds array. We will just need to add a note that the `_indexTroveManager` param is referring to index in `activeBranchIds` array, not the `branchId`.